internal/routes/all_routes.go

```diff
@@ -36,6 +45,10 @@ func apiRouter() http.Handler {
 		})
 	})
 
+	r.Get("/openapi.json", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is that necessary to gate behind Identity enforcement and should it live under the path prefix?

Reply:
This should not be gated indeed, just put this to the root namespace.

Reply:
the docs are metadata mainly for devs, that's why I was thinking to gate it behind the identity